tests/testAccess.js

```diff
@@ -305,6 +305,26 @@ describe('Access control', function() {
         });
     };
 
+    const getAssessmentInstanceCountdown = function(cookies, expectedStatusCode, callback) {
```

Comment:
I have a stashed version that merges this functionality into `getAssessmentInstance()` via `options` if that's preferred.

Reply:
I guess I'd be inclined to change `getAssessmentInstance()` to return the page via `callback(null, page)`. Then the `should not contain countdown timer` test could call the regular `getAssessmentInstance()` function and do the `elemList = $('#countdownDisplay');` testing inside that particular test. This would remove the need for the special `getAssessmentInstanceCountdown()` function.

Reply:
Not sure if this is what you had in mind: 1706b2f0d0d0feaeb7bf9e1ccdcb84f304789354

Reply:
Actually I guess you were thinking to leave the `statusCode` check inside `getAssessmentInstance()` 🤦

Reply:
Take 2: 0792d77264fa6832170e1190512199cadbcb5757 (I realized `page` is actually global, so we can just do the timer check via callback)

Reply:
Looks great! One tiny change below. I also hadn't realized that `page` was global. That makes things easy!

Reply:
Done